test/suite/test_truncate05.py

```diff
@@ -31,6 +31,7 @@
 
 # test_truncate05.py
 # Test various fast truncate visibility scenarios
+@wttest.skip_for_hook("disagg", "fast truncate is not supported yet")
```

Comment:
is there an easier wya to check if somehting does fast truncate instead of skipping tests? Also, can we attach the fixme comment here?